### DISCUSSION TITLE
A USER NOT a Programmer!

### DISCUSSION THREAD

**Discussion by LxCee** [community/community#21754] [New to GitHub]
<p>Greetings from the <strong>Great</strong> Country of <em><strong>TEXAS</strong></em>,</p>
<p>I keep getting told to read the README.TXT file in projects yet cannot find where that is located.  Even did a search on the “Search the Community” with zero (0), none, ziltch located. </p>
<p>Not looking for a project just some assistance in the USE of one of your programs titled “Password Maker Pro v0.8.5.2” so it is USABLE.  At this time and place NOTHING seems to work at all.  I have used the original version created by Eric many a year ago.  From the very beginning . . . and now seems to be the very ENDING of that once GREAT program, unless we can get together to share at leaat my issues.  Not going into detail at this time because there are TTTOOO many . . .</p>
<p>Thank you for reading my post and inform me it we can discuss this one project and where?</p>
<p>TIA, CU L8R, ‘e’ LxCee “LoneWanderer”</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/lxcee">@lxcee</a></p>
<p>It sounds like you may be looking for help with a project <em>hosted</em> on GitHub. We don’t actually have any affiliation with the Password Maker Pro product or service. We’re only able to provide support for GitHub itself.</p>
<p>You noted that you need help with Password Maker Pro. This may be the repository you need to check:</p>
<p><a href="https://github.com/passwordmaker/chrome-passwordmaker" rel="nofollow noopener">https://github.com/passwordmaker/chrome-passwordmaker</a></p>
<p>Additionally, it looks like they may offer support via issues in their repo:</p>
<p><a href="https://github.com/passwordmaker/chrome-passwordmaker/issues" rel="nofollow noopener">https://github.com/passwordmaker/chrome-passwordmaker/issues</a></p>
<p>I’d recommend checking those places and maybe opening an issue in that repo to ask for help directly from the project maintainers.</p>
<p>I hope this helps! </p>

**Comment by LxCee** [comment]
<p>“nadiajoyce.”</p>
<p>Please excuse me for not getting back to this post, did not remember doing it, just too many things to keep uo with.  I do <strong>THANK</strong>  <strong>YOU</strong> for the the reply.  As I had posted, not a programmer so will not be back to bother this COMMUNITY any.</p>
<p>TIA, CU L8R, ‘e’ LxCee “LoneWanderer”</p>
